contracts/test/behaviour/strategy.js

```diff
@@ -0,0 +1,336 @@
+
+        for (const asset of assets) {
+          const platformAddress = await strategy.assetToPToken(asset.address);
+          const withdrawAmount = await units("10000", asset);
+          await expect(tx)
+            .to.emit(strategy, "Withdrawal")
+            .withArgs(asset.address, platformAddress, withdrawAmount);
+          await expect(tx).to.emit(asset, "Transfer").withNamedArgs({
+            to: vault.address,
+            // FraxETHStrategy withdraws directly from the sfrxETH vault and not the strategy
+            // from: strategy.address,
+          });
+        }
+      });
+    });
+    it("Should allow transfer of arbitrary tokens by Governor", async () => {
+      const { strategy, usdc, weth, governor } = context();
+
+      const strategySigner = await impersonateAndFund(strategy.address);
+
+      // Add some USDC to the strategy
+      const usdcAmount = usdcUnits("987");
+      await usdc.connect(strategySigner).mint(usdcAmount);
+      // Governor can take the USDC from the strategy
+      const usdcTx = await strategy
+        .connect(governor)
+        .transferToken(usdc.address, usdcAmount);
+      await expect(usdcTx)
+        .to.emit(usdc, "Transfer")
+        .withArgs(strategy.address, governor.address, usdcAmount);
+
+      // Add some WETH to the strategy
+      const wethAmount = parseUnits("123");
+      await weth.connect(strategySigner).mint(wethAmount);
+      // Governor can take the WETH from the strategy
+      const wethTx = await strategy
+        .connect(governor)
+        .transferToken(weth.address, wethAmount);
+      await expect(wethTx)
+        .to.emit(weth, "Transfer")
+        .withArgs(strategy.address, governor.address, wethAmount);
+    });
+    it("Should not allow transfer of arbitrary token by non-Governor", async () => {
+      const { strategy, weth, strategist, matt, harvester, vault } = context();
+
+      const vaultSigner = await impersonateAndFund(vault.address);
+      const harvesterSigner = await impersonateAndFund(harvester.address);
+
+      for (const signer of [strategist, matt, harvesterSigner, vaultSigner]) {
+        await expect(
+          strategy.connect(signer).transferToken(weth.address, parseUnits("8"))
+        ).to.be.revertedWith("Caller is not the Governor");
+      }
+    });
+    it("Should allow the harvester to be set by the governor", async () => {
+      const { governor, harvester, strategy } = context();
+      const randomAddress = Wallet.createRandom().address;
+
+      const tx = await strategy
+        .connect(governor)
+        .setHarvesterAddress(randomAddress);
+      await expect(tx)
+        .to.emit(strategy, "HarvesterAddressesUpdated")
+        .withArgs(harvester.address, randomAddress);
+    });
+    it("Should not allow the harvester to be set by non-governor", async () => {
+      const { strategy, strategist, matt, harvester, vault } = context();
+      const randomAddress = Wallet.createRandom().address;
```

Comment:
nit: Could just make it `address(0x1)` or something instead

Reply:
That's an option but I like throwing a little randomness in